Algebraic Topology/algebraic-topology.tex

```diff
@@ -156,7 +156,7 @@ \section{Homotopy}
 \mydate{d2}{20}{1}{2017}
 
 \begin{defn}
-  A \de{category} is a directed graph.  Each vertex is called an
+  A \de{category} is a directed graph. Each vertex is called an %
```

Comment:
If we are going to define a category, we should use the correct definition. Otherwise, if we are trying to just get the idea across, maybe we should not phrase it as a definition?